ontology/yaml/resources/METERS/entity_types/FM .yaml

```diff
@@ -0,0 +1,24 @@
+# Copyright 2020 Google LLC
```

Comment:
Please remove the trailing space in the file name.

Reply:
Done